VirtoCommerce.WebhooksModule.Data/Models/WebhookEntity.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.WebHooksModule.Core.Models;
+
+namespace VirtoCommerce.WebhooksModule.Data.Models
+{
+    public class WebHookEntity : AuditableEntity
+    {
+        public WebHookEntity()
+        {
+            Events = new NullCollection<WebHookEventEntity>();
+        }
+
+        [StringLength(1024)]
+        public string Name { get; set; }
+        [StringLength(2083)]
```

Comment:
2083 - seems like a magic number, seems it is only for old IE. Let's make it 2048.